### ISSUE TITLE
Cant log on to patreon using google

### ISSUE THREAD

**Issue by AL-Jassim** [futo-org/grayjay-android#75]
When trying to sign into patreon using google account it just shows a white page, im guessing its google not allowing embedded browser sign in? No idea why it just shows a white screen but allows other log in methods to be used

**Comment by teaalltr** [comment]
Same problem for me, can't get past the white page after Google login

**Comment by teaalltr** [comment]
https://stackoverflow.com/questions/51167203/android-google-login-not-working-inside-webview here is the likely solution @Kelvin-FUTO

**Comment by AL-Jassim** [comment]
> Same problem for me, can't get past the white page after Google login

hello , i found a weird work around that i didnt think would work. go to your patreon account and change the password. i used the "i forgot my passoword" even though i could access the account through google, i thought maybe i can trick it into allowing me to login without google but keep my account and everything in it. it worked! give that a try, you can log on as a normal patreon account and as a google account if you wanted

**Comment by teaalltr** [comment]
@AL-Jassim thank you, the workaround works a treat!

**Comment by Kelvin-FUTO** [comment]
Correct, you can have a Patreon google account with a Patreon password.
We are already aware why Patreon Google login (Or any other non-Youtube google login) doesn't work, but at a glance it might be more non-trivial than it seems, hence it has not gotten priority.

**Comment by teaalltr** [comment]
Thanks @Kelvin-FUTO for the feedback!
